panoptes_aggregation/tests/reducer_tests/test_subject_difficulty_reducer.py

```diff
@@ -0,0 +1,83 @@
+from panoptes_aggregation.reducers.subject_difficulty_reducer import subject_difficulty_reducer
+from .base_test_class import ReducerTest
+
+
+def process(data):
+    return data
+
+
+extracted_data = [
+    {
+        'feedback':
+            {
+                "success": [
+                    False,
+                    False,
+                    False
+                ]
+            }
+    },
+    {
+        'feedback':
+            {
+                "success": [
+                    True,
+                    True,
+                    True
+                ]
+            }
+    },
+    {
+        'feedback':
+            {
+                "success": [
+                    True,
+                    True,
+                    True
+                ]
+            }
+    },
+    {
+        'feedback': None
```

Comment:
As stated in another comment, this can't happen with the way the code is written, the `feedback` key would not be on the extract if the return value is `None`.  This should be an empty dictionary to test the code correctly.